docs/guides/vtex-io/Storefront-Guides/getting-started-with-storefront-solutions.md

```diff
@@ -0,0 +1,46 @@
+| **Apps Admin (Extension Hub)** | There is no integration with the VTEX app platform. | Allows installation and use. | Allows installation and use of some apps. |
+| **Customizations** | Performance-focused: Easier to add customizations without degrading performance. | Composability-focused: Flexible and customizable through VTEX IO apps. | Customizations are limited and can be managed by creating, organizing, and editing the structure of folders, files, and components within the Layout feature. |
+| **Performance** | Maximum performance possible, as it uses [Jamstack](https://jamstack.org/), which serves pre-rendered content to a CDN. It is made dynamic through APIs and serverless functions. | SSR (Server Side Rendering) features generate the complete HTML of a page on the server as a response to a page request, while SPA (Single Page Application) loads elements such as the header only once the page is loaded. | Outdated technologies make it challenging to maintain optimal performance, especially in scenarios demanding multiple customizations. |
+| **Availability** | Installation is available upon request through [VTEX Support](https://help.vtex.com/en/support). For information on unavailable features, refer to the \[Features and Functionalities\](#features-and-funcionalities) topic. | General availability. | No longer available to new stores starting with VTEX. |
+| **Support**<br><br>_To learn more about our support, see our documentation [Responsibilities within the VTEX ecosystem](https://help.vtex.com/en/tutorial/responsabilidades-no-ecossistema-vtex--3vL9aWICDr3WR64DYi1fJ3)._ | Available for VTEX core commerce services, [FastStore packages](https://developers.vtex.com/docs/guides/faststore/docs-what-is-faststore), and troubleshooting code issues. | Available for VTEX core commerce services, VTEX IO native apps or components, and troubleshooting code issues. | Available for [VTEX core commerce services](https://developers.vtex.com/docs/guides/getting-started) and VTEX native [Controls](https://developers.vtex.com/docs/guides/list-of-controls-for-templates) and [Templates](https://help.vtex.com/en/tutorial/what-are-templates--4l7BQBYO9ycumsqua2CU88). |
+
+## Features and functionalities
+
+The VTEX platform is both composable and complete, which delivers a set of services that can be integrated to empower your ecommerce operation and enables customizations and integrations with third-party services. 
+
+However, the different storefront solutions offer varying levels of composability and extensibility. Below, you can explore the features and functionalities available for each storefront technology.
+
+|     |     |     |     |
+| --- | :---: | :---: | :---: |
+|     | **FastStore** | **Store Framework** | **Legacy CMS Portal** |
+| **GitHub integration** | ✅   | ✅   | ❌   |
+| **Support to VTEX IO backend apps** | ✅   | ✅   | ❌   |
```

Comment:
Legacy CMS doesn't support VTEX IO backend apps? 👀

Reply:
In Legacy stores, to run VTEX IO scripts it is necessary to develop a [Scripts app](https://developers.vtex.com/docs/guides/vtex-io-documentation-running-io-scripts-in-non-io-vtex-stores). Should I mention it here?

Reply:
Eu acho que pode remover essa linha porque algumas funcionalidades de back podem funcionar independentemente do front